src/smc-webapp/project.coffee

```diff
@@ -168,8 +173,6 @@ class ProjectPage
                             warning_banner_hidden.push(@project_id)
                             box.hide()
                         box.show()
-                    else
-                        box.hide()
```

Comment:
I'm not sure, but IIRC, that last `else box.hide()` has the use, that it hides a maybe visible box, when someone comes back to the project after upgrading. Maybe something has changed, but it had a purpose.